engines/router/missionctl/instrumentation/prometheus.go

```diff
@@ -32,6 +32,8 @@ var requestLatencyBuckets = []float64{
 	2000, 5000, 10000, 20000, 50000, 100000,
 }
 
+var additionalRegisteredMetricNames = set.New(nil)
```

Comment:
dont have full context of the code, would having `registeredMetrics` with `ExperimentEngineRequestMs`, `RouteRequestDurationMs` and `TuringComponentRequestDurationMs` be a safer option, in case these get registered again?

Reply:
True, that's a good point, but then again those are Router metrics and if any of the plugins attempts to register those metric names (for themselves), I'd prefer the Router to panic to let the plugin user/builder know that they are attempting an illegal operation instead of just pretending nothing's happening.